coordinator/statement_executor.go

```diff
@@ -41,10 +44,10 @@ type StatementExecutor struct {
 	MaxSelectBucketsN int
 }
 
-func (e *StatementExecutor) ExecuteStatement(stmt influxql.Statement, ctx *influxql.ExecutionContext) error {
+func (e *StatementExecutor) ExecuteStatement(stmt influxql.Statement, ctx influxql.ExecutionContext) error {
```

Comment:
This was as a result of a conversation with @dgnorton where we were considering the ability of the `StatementExecutor` to modify the context in some cases. We thought passing it by value would make it easier for a `StatementExecutor` to do that, although it's also possible by just doing this:

```
newCtx := *ctx
```